fulltext/backends/__json.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from six import StringIO
+from six import string_types
+from six import integer_types
+
+
+def _to_text(text, obj):
+    if isinstance(obj, dict):
+        for key in sorted(obj.keys()):
```

Comment:
consider sorting in place (`obj.sort()`); will save some time/mem if the json file is big

Reply:
I am not sure. `keys()` is a generator, `sorted()` returns a list. There is no `dict.sort()`. Thus my other option is something like:

```python
keys = list(obj.keys())
keys.sort()
for key in keys:
    ...
```

Which should be equivalent.

Reply:
Oh right, I forgot keys() on python 3 returns a generator.